cmd/state-svc/manager.go

```diff
@@ -144,7 +144,7 @@ func (s *serviceManager) CheckPid(pid int) (*int, error) {
 		exe, err := p.Exe()
 		if err != nil {
 			logging.Error("Could not detect executable for pid, error: %s", errs.JoinMessage(err))
-		} else if filepath.Base(exe) != constants.ServiceCommandName && strings.Contains(strings.ToLower(exe), "activestate") {
+		} else if !strings.HasPrefix(strings.ToLower(filepath.Base(exe)), constants.ServiceCommandName) {
```

Comment:
This is exactly the line of code I'm working on for https://activestatef.atlassian.net/browse/DX-738 😆